articles/proofs-arguments-and-zero-knowledge/chapter_12.md

```diff
@@ -0,0 +1,276 @@
+---
+Title: "Σ-Protocols and Commitments from Hardness of Discrete Logarithm"
+Status: Done
+Level: "3"
+---
+
+# Chapter 12: $Σ$-Protocols and Commitments from Hardness of Discrete Logarithm
+
+## 12.1 Cryptographic Background
+
+### 12.1.1 A Brief Introduction to Groups
+
+This is the definition of [group](../../terms/group.md).
+
+### 12.1.2 The Discrete Logarithm Problem and Background on Elliptic Curves
+
+#### 12.1.2.1 Discrete Log Problem
```

Comment:
Already have a section for this in discrete_logarithm.md

Reply:
Merge this with it

Reply:
Should I add the text that implies "discrete logarithm problem are not post-quantum secure" to discrete_logarithm.md?